projects/common/c5soc/c5soc_system_qsys.tcl

```diff
@@ -156,86 +156,76 @@ add_connection sys_clk.clk_reset sys_int_mem.reset1
-
-# display (vga-out-data)
-
-add_instance vga_out_data alt_vip_itc
-set_instance_parameter_value vga_out_data {H_ACTIVE_PIXELS} {1360}
-set_instance_parameter_value vga_out_data {V_ACTIVE_LINES} {768}
-set_instance_parameter_value vga_out_data {BPS} {8}
-set_instance_parameter_value vga_out_data {NUMBER_OF_COLOUR_PLANES} {4}
-set_instance_parameter_value vga_out_data {COLOUR_PLANES_ARE_IN_PARALLEL} {1}
-set_instance_parameter_value vga_out_data {ACCEPT_COLOURS_IN_SEQ} {0}
-set_instance_parameter_value vga_out_data {INTERLACED} {0}
-set_instance_parameter_value vga_out_data {USE_EMBEDDED_SYNCS} {0}
-set_instance_parameter_value vga_out_data {AP_LINE} {0}
-set_instance_parameter_value vga_out_data {ANC_LINE} {0}
-set_instance_parameter_value vga_out_data {H_BLANK} {0}
-set_instance_parameter_value vga_out_data {V_BLANK} {0}
-set_instance_parameter_value vga_out_data {H_SYNC_LENGTH} {112}
-set_instance_parameter_value vga_out_data {H_FRONT_PORCH} {64}
-set_instance_parameter_value vga_out_data {H_BACK_PORCH} {256}
-set_instance_parameter_value vga_out_data {V_SYNC_LENGTH} {6}
-set_instance_parameter_value vga_out_data {V_FRONT_PORCH} {3}
-set_instance_parameter_value vga_out_data {V_BACK_PORCH} {18}
-set_instance_parameter_value vga_out_data {F_RISING_EDGE} {0}
-set_instance_parameter_value vga_out_data {F_FALLING_EDGE} {0}
-set_instance_parameter_value vga_out_data {FIELD0_V_RISING_EDGE} {0}
-set_instance_parameter_value vga_out_data {FIELD0_ANC_LINE} {0}
-set_instance_parameter_value vga_out_data {FIELD0_V_BLANK} {0}
-set_instance_parameter_value vga_out_data {FIELD0_V_SYNC_LENGTH} {0}
-set_instance_parameter_value vga_out_data {FIELD0_V_FRONT_PORCH} {0}
-set_instance_parameter_value vga_out_data {FIELD0_V_BACK_PORCH} {0}
-set_instance_parameter_value vga_out_data {FIFO_DEPTH} {1920}
-set_instance_parameter_value vga_out_data {THRESHOLD} {1919}
-set_instance_parameter_value vga_out_data {CLOCKS_ARE_SAME} {0}
-set_instance_parameter_value vga_out_data {USE_CONTROL} {0}
-set_instance_parameter_value vga_out_data {GENERATE_SYNC} {0}
-set_instance_parameter_value vga_out_data {NO_OF_MODES} {1}
-set_instance_parameter_value vga_out_data {STD_WIDTH} {1}
-add_connection vga_pll.outclk0 vga_out_data.is_clk_rst
-add_connection sys_clk.clk_reset vga_out_data.is_clk_rst_reset
-add_connection vga_frame_reader.avalon_streaming_source vga_out_data.din
-add_interface vga_out_data conduit end
-set_interface_property vga_out_data EXPORT_OF vga_out_data.clocked_video
+add_instance pixel_clk_pll altera_pll
+set_instance_parameter_value pixel_clk_pll {gui_device_speed_grade} {2}
+set_instance_parameter_value pixel_clk_pll {gui_reference_clock_frequency} {50.0}
+set_instance_parameter_value pixel_clk_pll {gui_use_locked} {0}
+set_instance_parameter_value pixel_clk_pll {gui_number_of_clocks} {2}
+set_instance_parameter_value pixel_clk_pll {gui_output_clock_frequency0} {65.00}
+set_instance_parameter_value pixel_clk_pll {gui_output_clock_frequency1} {100.0}
```

Comment:
Can you put in commit description why these frequencies were chosen ? Do they correspond to a certain resolution ? Should they be specific to the maximum supported resolution so that we know the design meets timing ?